### ISSUE TITLE
Need Help with pose capturing and pose matching technique

### ISSUE THREAD

**Issue by rohitshirur007** [tucan9389/PoseEstimation-CoreML#36]
hello @tucan9389 i'm flutter developer i don't understand swift language and i just want to know about how you captured the poses with only skeleton drawn and compared it with live feed i'm using different posenet moel here please guide me with this. 

Since this is not an issue so you can close this and please reply here email : shirurrohit@gmail.com 

Thanks :)

**Comment by tucan9389** [comment]
@rohitshirur007 
https://github.com/tucan9389/PoseEstimation-CoreML/pull/20
Here is the guide on how to implement the matching algorithm between two different pose joints. 
If you have more questions, please tell me.

**Comment by rohitshirur007** [comment]
Yes, Exactly what we're doing now is calculating angles for each poses so my question is are you giving all the points to only one calculation method or for each poses you're making different calculation?

( our Calculation includes calculating distance also)

Thanks :)
